nginx_proxy/vhost.mustache

```diff
@@ -68,7 +68,7 @@ server {
     client_max_body_size {{max_body_size}};
     {{/max_body_size}}
     location / {
-        proxy_pass {{#remote_ssl}}https{{#remote_ssl}}{{^remote_ssl}}http{{/remote_ssl}://{{#remote}}{{remote}}{{/remote}}{{^remote}}172.17.0.1{{/remote}}:{{port}};
+        proxy_pass {{#remote_ssl}}https{{#remote_ssl}}{{^remote_ssl}}http{{/remote_ssl}}://{{#remote}}{{remote}}{{/remote}}{{^remote}}172.17.0.1{{/remote}}:{{port}};
```

Comment:
Isn't there a second issue? I believe it should be `{{#remote_ssl}}https{{/remote_ssl}}`, no?

Reply:
Never looked at "mustache" before, but I suspect you're correct.  Included your suggestion.